src/index.js

```diff
@@ -1,15 +1,15 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
+/* eslint-disable */
+import React from "react";
+import ReactDOM from "react-dom";
+import { HashRouter  } from "react-router-dom";
+import TodoContainer from "/home/isaka/to-do/src/functionalBased/components/TodoContainer.js";
+import "/home/isaka/to-do/src/functionalBased/App.css";
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <HashRouter>
+      <TodoContainer />
+    </HashRouter>
```

Comment:
You did a really great job adding HashRouter but there is a problem because it stores the current location in the hash portion of the URL. The URL you get here is not so clean so please change it to Router and Switch components.

Reply:
HI @Donard97👋 , thanks for the suggestion i fixed this issue already I used **BrowserRouter now**